go.mod

```diff
@@ -26,7 +27,7 @@ require (
 	github.com/docker/go-units v0.4.0
 	github.com/emicklei/go-restful v2.9.5+incompatible
 	github.com/fsnotify/fsnotify v1.4.9
-	github.com/gogo/googleapis v1.4.0
+	github.com/gogo/googleapis v1.4.1
```

Comment:
Was the update needed to make this work? Otherwise I tend to prefer to keep it separate

Reply:
Actually, this looks incorrect, because we have a `replace` rule in place, so from the _actual_ code, this is a "no-op" on our side, but will affect other users of containerd.

We need to add some comments for these (https://github.com/containerd/containerd/pull/5208)

Reply:
switching back to `github.com/gogo/googleapis v1.4.0`